uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java

```diff
@@ -50,6 +50,10 @@ public BrowserChrome() {
     public void handleFirstRun() {
         Logger.i(TAG, "Handle First Run of Browser..");
         UiAutomatorUtils.handleButtonClick("com.android.chrome:id/terms_accept");
+        final UiObject nextButtonForLiteMode = UiAutomatorUtils.obtainUiObjectWithText("Next");
+        if (nextButtonForLiteMode.exists()){
+            UiAutomatorUtils.handleButtonClickForObjectWithText("Next");
+        }
```

Comment:
We can't put this change in...because this would mean that at worst case we are extending the time taken for each test by 30 seconds (we look for a UI element for 30 seconds max or less if we find it earlier)...so if a UI element is not there such as the lite mode button and in this case most devices (including ones on Firebase) won't have a lite mode in the version of Chrome that they are using...and so the button will never be found but we would still keep looking for it for 30 seconds before we realize that hey not found....I think we can optimize the performance of the tests by not doing this check or only doing this check on devices where we expect to see the lite mode button. 

I would suggest putting this logic under a boolean flag that we can toggle and based on the value of that flag we decide whether to look for the lite mode button or not. (as an advance thought...you may be able to toggle that flag based on geographical location of the device....you will have to check whether Android provides any APIs to support that)

Reply:
Instead of hard coding a false here....I suggest defining a `static final boolean` variable at the top of this class and set it to false there...and then use the value here

Reply:
Apply change here d21cd2bbfe809f425adf0d4bf186efe8bd91cdad